### ISSUE TITLE
Explainers for APIs with a history should mention that history

### ISSUE THREAD

**Issue by jyasskin** [w3ctag/w3ctag.github.io#22]
I'm reminded about this by Tantek's complaint in https://tantek.com/2020/026/t2 that https://github.com/WICG/contact-api doesn't mention the long history of failed Contacts API proposals. https://w3ctag.github.io/explainers should probably suggest that authors include a list or description of previous overlapping proposals. These might go in the "Considered alternatives" or "References & acknowledgements" sections depending on how much their ideas were re-used or rejected.

**Comment by hober** [comment]
Indeed; this was one of the first requests from the TAG on design-reviews#337, which @travisleithead captured in [this comment from Feb 2019](https://github.com/w3ctag/design-reviews/issues/337#issuecomment-460555385)

> We note that this is not the first time the web has seen a proposal for a Contact picker. Sadly, the previous efforts have not gained traction for various reasons. We would love to see some background on what differentiates this proposal (e.g., how will this proposal capture implementer interest and why?)
